serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -57,55 +164,162 @@ Double-check your `$\{HOME\}/.docker/config.json`. If you're using local desktop
+====
+
+Following a very basic `SonataFlowPlatform` Custom Resource example to work on Kubernetes:
+
+.Example of a basic SonataFlowPlatform on Kubernetes
+[source,yaml,subs="attributes+"]
 ----
-kubectl apply -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml -n kogito-workflows
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  platform:
+    buildStrategyOptions:
+      KanikoBuildCacheEnabled: "true"
+    registry:
+      address: quay.io/kiegroup <1>
+      secret: regcred <2>
 ----
 
-[NOTE]
+<1> Your registry address
+<2> The secret name created in the steps above
+
+On Minikube, you can remove the `registry` information entirely since you don't need credentials for pushing to the internal registry:
+
+.Example of a basic SonataFlowPlatform on Minikube
+[source,yaml,subs="attributes+"]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  platform:
+    buildStrategyOptions:
+      KanikoBuildCacheEnabled: "true"
+----
+
+[TIP]
 ====
-In this Custom Resource, `spec.platform.registry.secret` is the name of the Secret you created just before.
+The `spec.platform.buildStrategyOptions.KanikoBuildCacheEnabled` enables the internal Kaniko process to cache builder images to try to speed up the bulding execution.
 ====
 
-You can also update "on-the-fly" the `KogitoServerlessPlatform` registry field with this command (change <YOUR_REGISTRY>)
+You can save this file locally and run the following command:
 
-.Create a `KogitoServerlessPlatform` with a specific registry
+.Create a `SonataFlowPlatform`
 [source,bash,subs="attributes+"]
 ----
-curl https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml | sed "s|address: .*|address: <YOUR_REGISTRY>" | kubectl apply -f -
+kubectl apply -f my-sonataflowplatform.yaml -n workflows
 ----
 
-In order to retrieve the Cluster IP address of Minikube's internal registry to configure your platform, you can use the following command:
+You can also update "on-the-fly" the `SonataFlowPlatform` registry field with this command (change <YOUR_REGISTRY>)
 
-.Retrieve Minikube registry internal IP
+.Create a `SonataFlowPlatform` with a specific registry
 [source,bash,subs="attributes+"]
 ----
-kubectl get svc registry -n kube-system -ojsonpath='{.spec.clusterIP}'
+cat my-sonataflowplatform.yaml | sed "s|address: .*|address: <YOUR_REGISTRY>" | kubectl apply -f -
 ----
 
-== Build and deploy your Workflow application
+[#building-openshift]
+== Building on OpenShift
+
+You don't need to do anything to build on OpenShift since the operator will configure everything for you. There are a few customizations you can do described in the <<configure-build-system>> section.
```

Comment:
Just clarifying, so on OpenShift, we don't allow pushing to external registry as we support just `ImageStream`?

Reply:
That's correct. On OpenShift, users can configure their internal registry as they please. If I'm not mistaken, they can even mirror an external/enterprise one. That's why our build system is for limited scenarios, we won't stretch there since we are planning to create integrations with external build systems in the near future.